draft-ietf-keytrans-architecture.md

```diff
@@ -400,6 +400,94 @@ Alice                  Transparency Log                  Manager
 Transparency Log to the Manager. Rejected requests are blocked." }
 
 
+# Combining Logs
+
+There are many cases where it makes sense to operate multiple cooperating log
+instances, for example:
+
+- A service provider may decide that it's prudent to rotate its cryptographic
+  keys, or migrate to a new deployment mode. They can do this by creating a new
+  log instance with new cryptographic keys, operating under a new
+  deployment mode if desired, and migrating their data from the old log to the
+  new log while users are able to query both.
+- A service provider may choose to operate multiple logs to improve their
+  ability to scale or provide higher availability.
+- A federated system may allow each participant in the federation to operate
+  their own log for their own users.
```

Comment:
do you envision sharding to be used, by year for instance? I have the impression it provides a slightly more concrete example.